cmd/pro/provider/up.go

```diff
@@ -85,10 +112,41 @@ func (cmd *UpCmd) up(ctx context.Context, workspace *managementv1.DevPodWorkspac
 		return err
 	}
 
-	_, err = remotecommand.ExecuteConn(ctx, conn, cmd.streams.Stdin, cmd.streams.Stdout, cmd.streams.Stderr, cmd.Log.ErrorStreamOnly())
+	_, err = remotecommand.ExecuteConn(ctx, conn, cmd.streams.Stdin, cmd.streams.Stdout, cmd.streams.Stderr, cmd.Log)
 	if err != nil {
 		return fmt.Errorf("error executing: %w", err)
 	}
 
 	return nil
 }
+
+func templateUpdateRequired(instance *managementv1.DevPodWorkspaceInstance) bool {
+	var templateResolved, templateChangesAvailable bool
+	for _, condition := range instance.Status.Conditions {
+		if condition.Type == storagev1.InstanceTemplateResolved {
+			templateResolved = condition.Status == corev1.ConditionTrue
```

Comment:
does this only consider the last value? since we don't break. Are the status conditions in chronological order and we only care about the last value for type `InstanceTemplateResolved` and `InstanceTemplateSynced`?